CarRental/CarRental.Web/Models/RentalViewModel.cs

```diff
@@ -10,5 +12,49 @@ public class RentalViewModel
     public decimal? TotalCost { get; set; }
 
     public string? CustomerName { get; set; }
+
+    [Display(Name = "Car License Plate")]
     public string? CarLicencePlate { get; set; }
+
+
+    //private DateTime _beginDate;
+    //private DateTime _endDate;
+    //private decimal _totaCost;
+
+
+    //public DateTime BeginDate
+    //{
+    //    get
+    //    {
+    //        return _beginDate;
+    //    }
+    //    set
+    //    {
+    //        _beginDate = value < _endDate ? value : _endDate;
+    //    }
+    //}
+
+    //public DateTime EndDate
+    //{
+    //    get
+    //    {
+    //        return _endDate;
+    //    }
+    //    set
+    //    {
+    //        _endDate = value > _beginDate ? value : _beginDate;
+    //    }
+    //}
+
+    //public decimal? TotalCost
```

Comment:
o co chodzi z tymi komentarzami

Reply:
to zostało po ekperymencie; będzie do usunięcia